app-backup/deja-dup/deja-dup-40.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson eutils gnome2-utils vala xdg
+
+DESCRIPTION="Simple backup tool using duplicity back-end"
+HOMEPAGE="https://gitlab.gnome.org/World/deja-dup"
+SRC_URI="https://gitlab.gnome.org/World/${PN}/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="nautilus test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
```

Comment:
Given that the whole thing is written in vala, are you sure you don't need introspection flags everywhere?